go/ql/lib/semmle/go/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -228,6 +280,105 @@ module SourceSinkInterpretationInput implements
     }
   }
 
+  /**
+   * Gets a method or field spec for `e` which applies in the context of
+   * qualifier `qual`.
+   *
+   * Note that naively checking `e`'s qualified name is not correct, because
+   * `Method`s and `Field`s may have multiple qualified names due to embedding.
```

Comment:
I think I was wrong about this -- I think it's only Methods that can have multiple qnames like this, and field embedding is represented more explicitly
```suggestion
   * `Method`s may have multiple qualified names due to embedding.
```